.github/workflows/release.yml

```
@@ -48,8 +48,15 @@ jobs:
           # Get the latest CLI version tag dynamically
           LATEST_VERSION=$(git tag -l | grep -E '^[0-9]+\.[0-9]+\.[0-9]+$' | sort -V | tail -n1 || echo "0.0.0")
 
-          # Use git cliff to determine next version
-          NEXT_VERSION=$(git cliff --include-path "cli/**/*" --config cliff.toml --repository "../" --bumped-version 2>/dev/null -- ${LATEST_VERSION}..HEAD)
+          # Check if there are any releasable changes by generating changelog
+          CHANGELOG=$(git cliff --include-path "cli/**/*" --config cliff.toml --repository "../" 2>/dev/null -- ${LATEST_VERSION}..HEAD || echo "")
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)